sdk/plugin/grpc_system.go

```diff
@@ -470,3 +461,47 @@ func (s *gRPCSystemViewServer) GenerateIdentityToken(ctx context.Context, req *p
 		TTL:   int64(res.TTL.Seconds()),
 	}, nil
 }
+
+func (s *gRPCSystemViewServer) RegisterRotationJob(ctx context.Context, req *pb.RegisterRotationJobRequest) (*pb.RegisterRotationJobResponse, error) {
```

Comment:
Should we return an error if any of the required RegisterRotationJobRequest fields are not set? What happens if a plugin developer does not set MountType, for example?